src/04-modality-specific-files/04-intracranial-electroencephalography.md

```diff
@@ -0,0 +1,489 @@
+# intracranial Electroencephalography (iEEG)
+
+Support for iEEG was developed as a [BIDS Extension Proposal](../06-extensions.md#bids-extension-proposals).
+Please cite the following paper when referring to this part of the standard in
+context of the academic literature:
+
+> Holdgraf, C., Appelhoff, S., Bickel, S., Bouchard, K., D'Ambrosio, S.,
+> David, O., … Hermes, D. (2018, December 13). BIDS-iEEG: an extension to the
```

Comment:
It would be nice to list all of the co-authors, but this can be fixed with an followup PR.

Reply:
done.